azure/lb.sh

```diff
@@ -120,6 +120,25 @@ fn azure_lb_addresspool_delete(name, group, lbname) {
 	)
 }
 
+fn azure_lb_addresspool_get_id(addrpoolname, resgroup, lbname) {
+	out, err <= (
+		az network lb address-pool show
+					--resource-group $resgroup
+					--lb-name $lbname
+					--name $addrpoolname
+					--output json
+					>[2=]
```

Comment:
I don't know if we need to redirect stderr here. @tiago4orion what do you think?